ckanext/ssm/plugin.py

```diff
@@ -4,19 +4,30 @@
 """
 
 import re
+import requests
 import six
 
 from logging import getLogger
 
 import boto3
-from boto.utils import get_instance_identity
 
 from ckan.plugins import implements, SingletonPlugin, IConfigurer
 
 LOG = getLogger(__name__)
 SSM_PARAMETER_SYNTAX = re.compile(r'[$][{]ssm:(/[-a-zA-Z0-9_./]+)[}]')
 
 
+def get_instance_identity_document():
+    token = requests.put(
+        'http://169.254.169.254/latest/api/token',
+        headers={'X-aws-ec2-metadata-token-ttl-seconds': '60'}
```

Comment:
so is this calling the v2 version?

Reply:
Yes; there is no token interaction in v1.